### PR TITLE
Improved icon tooltip

### PR DESCRIPTION
**Pull request by airenzp**
## Description

Improved icon tooltip. Same concept may be applied to other tooltips if wanted. Can be tested in the profile icons or in the scatter plot.

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/src/style.css [hunk 1]
```diff
@@ -1253,3 +1253,30 @@ sandbox header
     from {transform: rotate(0deg);}
     to {transform: rotate(359deg);}
  }
+
+
+/* Tooltip container */
+.tooltip {
```
Comment (siosonel) [inline client/src/style.css:1259]: @airenzp this and other css classes below should be namespaced, `.sja_root_holder .tooltip`

**Comment by xzhou82** [comment]
[at barchart](http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22SJLife%22,%22genome%22:%22hg38%22,%22plots%22:[{%22chartType%22:%22summary%22,%22term%22:{%22id%22:%22diaggrp%22}}],%22nav%22:{%22activeTab%22:1}}), clicking menu btn the download icon no longer moves to the right of menu btn

File: client/src/style.css [hunk 2]
```diff
@@ -1253,3 +1253,30 @@ sandbox header
...
+    position: relative;
+    display: inline-block;
```
Comment (siosonel) [inline client/src/style.css:1261]: best to set `pointer-event: none` to minimize the chance that the invisible div would block other divs

**Comment by siosonel** [comment]
Should consider using css-only approach, no need to have an `addSvg()` function, so very minimal code changes. For example, see Carson's answer in https://stackoverflow.com/questions/9150796/change-how-fast-title-attributes-tooltip-appears:
```html
<style>
  [title] {
    position: relative;
  }

  [title]:after {
    content: attr(title);
    position: absolute;
    left: 50%;
    bottom: 100%; /* put it on the top */
    background-color: yellow;
    width: max-content;
    opacity: 0;
    -webkit-transition: opacity 0.75s ease-in-out; /* 👈 Change the time to meet your requirements. */
  }

  [title]:hover:after {
    opacity: 1;
  }
</style>

<div style="min-height:5rem"></div>
<div style="min-width: 5rem; border: 2px solid red;" title="hello world">my div</div>
<button title="for debug">button</button>
```

**Comment by airenzp** [comment]
I will close this PR to apply the idea suggested by @siosonel, explained in the link

**Pull request closed without merging**